packages/components/src/components/modal/modal.js

```diff
@@ -136,6 +145,12 @@ class Modal extends mixin(
         this.options.classBody,
         false
       );
+      if (this.options.selectorFocusOnClose || this.previouslyFocusedNode)
```

Comment:
Not sure why our ESLint didn't catch this, but we mandate curly braces for every `if()` block.